.NET/Microsoft.Recognizers.Text.DateTime/Hindi/Parsers/HindiDurationParserConfiguration.cs

```diff
@@ -8,8 +11,8 @@ public class HindiDurationParserConfiguration : BaseDateTimeOptionsConfiguration
         public HindiDurationParserConfiguration(ICommonDateTimeParserConfiguration config)
            : base(config)
         {
-            CardinalExtractor = config.CardinalExtractor;
-            NumberParser = config.NumberParser;
+            CardinalExtractor = Number.Hindi.CardinalExtractor.GetInstance();
```

Comment:
Why not get these two from the parser config?

Reply:
right, moved modifications to HindiCommonDateTimeParserConfiguration